### DISCUSSION TITLE
Reverse Proxy using Engintron? (Nginx + Apache)

### DISCUSSION THREAD

**Discussion by iantrudell** [strapi/strapi#7659] [Legacy Topics]
Hi, I'm using [Engintron ](https://engintron.com/) ([GitHub](https://github.com/engintron/engintron)) for my websites running on CentOS 7 with cPanel. I'm trying to configure Strapi to use it as a reverse proxy, but am running into difficulties since the configurations don't match the standard Nginx installation guide.

Here is the file structure for /etc/nginx/
![nginx_file_structure](https://user-images.githubusercontent.com/33202657/91736360-869a8600-eb7b-11ea-9390-fccbb642815a.PNG)

and here is the file structure for /etc/nginx/conf.d
![nginx_file_structure2](https://user-images.githubusercontent.com/33202657/91736418-9e720a00-eb7b-11ea-9e5b-3a38bdfdbc4b.PNG)

I started by adding upstream.conf in /etc/nginx/conf.d with this content:
```
# Strapi server
upstream strapi {
    server 127.0.0.1:1337;
}
```

But when I tried to follow the "Sub-Folder Split" instructions, I saw no "sites-available" or "sites-enabled" folders.

You can see all the contents of the standard setup for Engintron here:
https://github.com/engintron/engintron/tree/master/nginx

I'm not sure where to put the server blocks for Strapi, or if I should be looking at putting that section in Apache? Engintron seems to use both...

Additionally, I'm not sure what path to use for the SSL certificates since I'm using AutoSSL with cPanel, and the certificates seem to be regenerated frequently with hashes appended to the end for each domain.

Any help is much appreciated!

**Comment by derrickmehaffy** [comment]
Centos's default nginx isn't compiled as the "Debian version" that has default includes for sites-av* or sites-enabled.

In your `/etc/nginx/nginx.conf` you can include this line `include /etc/nginx/sites-enabled/*;` and just create the directories manually.
Same applies for the `/etc/nginx/conf.d` if there isn't an include option in the default config.
